shell/browser/api/electron_api_menu_mac.mm

```diff
@@ -156,7 +155,6 @@
   base::mac::ScopedSendingEvent sendingEventScoper;
 
   // Don't emit unresponsive event when showing menu.
```

Comment:
This can be removed too.

Reply:
See https://github.com/electron/electron/pull/35507